module/leftPanel/verticalisePanel/rectangularClockSubModule.js

```diff
@@ -32,17 +33,18 @@ var RectangularClockSubModule = class RectangularClockSubModule {
         this.indicatorPad.show();
     }
 
-    removeDotsFromClock() {
```

Comment:
thinking about this after the fact, I'm realizing that `removeDotsFromClock` could be interpreted in a couple different ways:

1. Remove the colon `:` character
2. Stop the clock from becoming truncated and ellipsized (end of the text cut off with `...`)

I assumed this function was trying to solve #2 
Let me know if this you actually meant #1 and this is an undesirable change.

Reply:
The goal of this function was indeed to remove the colon. I decided to split the clock on multiple line because of the lack of width available. We can't 4 digits in a row with the colons and the space needed to display it correctly.

Reply:
ah fair enough. I think I'll close this down and at least put up a new PR that makes the intent clearer.